.changeset/beige-cherries-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+"@soundxyz/sdk": patch
```

Comment:
```suggestion
"@soundxyz/sdk": major
```

Reply:
should this be major? minor most appropriate since nothing should be breaking

Reply:
https://github.com/soundxyz/sdk/pull/215#pullrequestreview-1407402258

Reply:
![image](https://user-images.githubusercontent.com/8672915/235420175-b0acd172-1aca-40cb-8b40-b6912f399751.png)

those imports will break, so yes

Reply:
that's the thing like when dealing with peer dependencies is always a tricky thing

Reply:
ok let's just rely on the peer dependencies warnings